flex-templates/python/regional_dlp_transform/bigquery_dlp_bigquery.py

```diff
@@ -28,45 +28,53 @@
 def run(argv=None, save_main_session=True):
     """Build and run the pipeline."""
     parser = argparse.ArgumentParser()
-    parser.add_argument(
+    group = parser.add_argument_group()
+    group_exclusive = parser.add_mutually_exclusive_group(required=True)
+    group_exclusive.add_argument(
+        '--query',
+        help=(
+            'Input query to retrieve data from Dataset. '
```

Comment:
add comment about specifying one or the other for BQ source and why a user would pick one vs the other